docs/sources/_index.md

```diff
@@ -33,6 +33,14 @@ Writers’ Toolkit is for anyone who writes or edits customer-facing technical d
 
 We write our documentation using U.S. English.
 
+Our style guide is based on [Google's developer documentation style guide](https://developers.google.com/style). If you can't find guidance on a specific topic in the Writers' Toolkit, refer to Google's style guide.
```

Comment:
nit: I believe it should be just "Writers' Toolkit" without "the".

 ```suggestion
Our style guide is based on [Google's developer documentation style guide](https://developers.google.com/style). If you can't find guidance on a specific topic in Writers' Toolkit, refer to Google's style guide.
```

Reply:
Good catch, thanks! 🙏